src/ansys/simai/core/data/optimizations.py

```diff
@@ -239,16 +341,10 @@ def get(self, trial_run_id: str):
         """Get a specific trial run from the server."""
         return self._model_from(self._client._api.get_optimization_trial_run(trial_run_id))
 
-    def _try_geometry(
-        self,
-        optimization: Identifiable[Optimization],
-        geometry: Identifiable[Geometry],
-        geometry_parameters: Dict,
+    def _run_iteration(
```

Comment:
nit: same method name for both optim and optimtrialrun objects, but with different args..